pkg/bgpv1/manager/reconciler/service.go

```diff
@@ -24,24 +25,34 @@ import (
 	"github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/labels"
 	slim_metav1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
 	"github.com/cilium/cilium/pkg/loadbalancer"
+	"github.com/cilium/cilium/pkg/option"
 	ciliumslices "github.com/cilium/cilium/pkg/slices"
 )
 
+type RoutesConfig struct {
```

Comment:
This config seems to be defined at 2 places now. What about defining a common `pkg/bgpv1/config` and using it from there? In the future, we can move there other BGP-specific config options, such as `bgp-secrets-namespace`

Reply:
sounds good